WordPress/Classes/ViewRelated/Me/App Settings/AppSettingsViewController.swift

```diff
@@ -343,13 +343,15 @@ class AppSettingsViewController: UITableViewController {
             let viewController = SettingsSelectionViewController(dictionary: settingsSelectionConfiguration)
 
             viewController?.onItemSelected = { (section: Any!) -> () in
+                let oldDefaultSection = MySiteSettings().defaultSection
                 guard let section = section as? Int,
-                    let defaultSection = MySiteViewController.Section(rawValue: section) else {
+                      let newDefaultSection = MySiteViewController.Section(rawValue: section),
+                        newDefaultSection != oldDefaultSection  else {
```

Comment:
[nit] alignment and extra space before `else`
```suggestion
                      let newDefaultSection = MySiteViewController.Section(rawValue: section),
                      newDefaultSection != oldDefaultSection else {
```